code/game/machinery/rechargestation.dm

```diff
@@ -87,6 +87,7 @@
 	return 1
 
 /obj/structure/machinery/recharge_station/stop_processing()
+	LAZYREMOVE(GLOB.hearing_objects, src)
```

Comment:
Removing here means we will only remove it from the hearing objects once this charger fully recharges and still has no occupant. Should someone re-enter prior to this, you'll LAZYADD again without the LAZYREMOVE occurring.